SimPEG/Maps.py

```diff
@@ -373,6 +383,254 @@ def nP(self):
         return self._nP
 
 
+class SelfConsistentEffectiveMedium(IdentityMap, properties.HasProperties):
```

Comment:
this should eventually be pulled out, but at this point, I would rather address that in a separate pr. thoughts @rowanc1?